core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -540,8 +540,13 @@ private ExecutionModel parseCreateTable(GenericLexer lexer, SqlExecutionContext
         tok = tok(lexer, "'(' or 'as'");
 
         if (Chars.equals(tok, '(')) {
-            lexer.unparseLast();
-            parseCreateTableColumns(lexer, model);
+            tok = tok(lexer, "like");
+            if (isLikeKeyword(tok)) {
+                parseLikeTableName(lexer, model);
```

Comment:
We should return from the method here unless we want to allow users to call DDLs like `CREATE TABLE dst (LIKE srt), INDEX(col_1) TIMESTAMP(col_2) with maxUncommittedRows = 42, commitLag = 42s;`. If we want to support such overrides for index, designated timestamp and parameters, we'd have to add more tests to cover that. But I'd propose to keep things simple for now and support only the basic syntax, i.e. `CREATE TABLE dst (LIKE srt);`.

Reply:
Now returning model after method call parseLikeTableName